Elastiflix/docker-compose-elastic-otel.yml

```diff
@@ -96,6 +96,7 @@ services:
     ports:
       - "5003:5000"
     environment:
+      - ELASTIC_APM_SECRET_TOKEN=${ELASTIC_APM_SECRET_TOKEN}
```

Comment:
[for reviewer] for now the "manual" variant relies on this environment variable to explicitly deal with authentication, this is something that could be improved later to only rely on standard otel environment variables.